pom.xml

```diff
@@ -1,4 +1,5 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!-- do_not_remove: published-with-gradle-metadata -->
```

Comment:
Quick question here: is this used by a tool, or just informational? Is there a link to something that explains it (could be in different comment if so)? Would be nice to have something for curious minds :)

Reply:
Yes. This line is discovered by Gradle so that it knows that a `.module` file was published.

If you publish with Gradle, where the POM is generated, you actually get more explanation on the comment. I now added the whole explanation here as well.